security/cmd/istio_ca/main.go

```diff
@@ -262,6 +263,16 @@ func init() {
 	cmd.InitializeFlags(rootCmd)
 }
 
+func initEnvVars() {
+	selfNamespace := env.RegisterStringVar("POD_NAMESPACE", "",
```

Comment:
Sorry, why do we need the `selfNamesapce`? Isn't it always the same as `istioCaStorageNamespace`?

Reply:
didn't realize `istioCaStorageNamespace` existed, will remove `POD_NAMESPACE`

Reply:
Looked a bit more into this -- from the comment on the CLI flag, `istioCaStorageNamespace` is the "namespace where the Citadel pod is running. Will not be used if explicit file or other storage mechanism is specified". Just to make sure: this will always be set to the namespace where the pod is running in, even if a different storage mechanism is used, correct?